src/Node/ClassMethod.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Node;
+
+class ClassMethod
+{
+
+	public function __construct(
+		private \PhpParser\Node\Stmt\ClassMethod $node,
+		private bool $isDeclaredInTrait,
+	)
+	{
+	}
+
```

Comment:
```suggestion
```

Reply:
OH BUT WHAT ABOUT MY NEWLINE

Reply:
I didn't understand that :) There should be a PHPCS sniff catching that (https://github.com/slevomat/coding-standard#slevomatcodingstandardclassesmethodspacing-), feel free to enable that and send a new PR :)

Reply:
this is really bothering me, so yes, I'll do that :)